app/assets/stylesheets/users.scss

```diff
@@ -1,3 +1,86 @@
 // Place all the styles related to the Users controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: https://sass-lang.com/
+
```

Comment:
Is this file still necessary with #158 merged?

Reply:
Yes, it is. We are using some styles for the search form.